specs/TrackingPrevention.md

```diff
@@ -0,0 +1,205 @@
+Tracking Prevention
+===
+
+# Background
+The WebView2 team has been asked for an API to toggle tracking prevention and also ability to control levels
+of tracking prevention.
+
+We are proposing two API's
+IsTrackingPreventionEnabled: This API allows you to enable/disable the tracking prevention feature. By default
+tracking prevention feature is enabled and set to `Balanced` level for WebView2. You can set the
+`CoreWebView2EnvironmentOptions.IsTrackingPreventionEnabled` property to false to disable the tracking prevention

```

Comment:
Name brainstorming welcome.  I tried to avoid double negative but not sure if "Prevention" is a negative in the same way "prevent" is.  How do we feel about "default value true"?  
(I tend to default Boolean properties to false so that "opt-in" to non-default -> "turning On optional behavior" -> set something to True, and it's aligned to the habit that zero-fill is a default value, but it's not a semantic requirement of properties.)

Reply:
Let's do `EnableTrackingPrevention` default to `true`

Also considered:
* PreventTracking
* AllowTrackingPrevention